src/main/java/com/toptrumps/cli/TopTrumpsCLIApp.java

```diff
@@ -1,18 +1,15 @@
+package com.toptrumps.cli;
 
-
-package commandline;
-import core.*;
-
+import com.toptrumps.core.Deck;
 
 /**
  * Top Trumps command line application
  */
-public class TopTrumpsCLIApplication {
```

Comment:
Are you sure renaming this is a good idea if they use automated tests?

Reply:
We can double-check with the professor in the next lecture. But anyway, they were not crystal clear in the task document that we couldn't rename the classes.

> After submission your program will be tested, so it is particularly important that your program does not refer to the absolute location of the deck data file. When testing your program within Eclipse, store the deck data file in the current working directory.

Reply:
@hughwin I spoke with the lecturer and he said we're fine! We just need to provide a mapping table on our report :)